scripts/default.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Yeah. just something to play with... Do we have anything to start with script wise?